themes/mlibrary_new/items/show.php

```diff
@@ -154,34 +145,14 @@
       </dl>
       </div>
 </div>
-<?php } ?>
-
-<div class="col-md-6 col-md-offset-3">
-  <h2 class="metadata--heading">Item Data</h2>
-  <div class="detail-nav-border"></div>
-  <dl class="metadata--list">
-    <dt>Creator</dt>
-    <dd><?php print $creator; ?></dd>
-    <dt>Date</dt>
-    <dd><?php print $date; ?></dd>
-    <dt>Source</dt>
-    <dd><?php print $source; ?></dd>
-    <dt>Description</dt>
-    <dd><?php print $description; ?></dd>
-    <dt>Publisher</dt>
-    <dd><?php print $publisher; ?></dd>
-    <dt>Contributor</dt>
-    <dd><?php print $contributor; ?></dd>
-    <dt>Language</dt>
-    <dd><?php print $language; ?></dd>
-    <dt>Type</dt>
-    <dd><?php print $type; ?></dd>
-    <dt>Format</dt>
-    <dd><?php print $format; ?></dd>
-    <dt>Rights</dt>
-    <dd><?php print $rights; ?></dd>
-  </dl>
+<?php } 
+if (isset($item_metadata)) {?>
```

Comment:
`$item_metadata` will always be set, do you mean to check `!empty($item_metadata)` ?